crates/base/src/deno_runtime.rs

```diff
@@ -435,7 +438,13 @@ impl DenoRuntime {
                 Err(err) => return Poll::Ready(Err(err)),
             };
 
-            let poll_result = js_runtime.poll_event_loop(cx, false);
+            let poll_result = js_runtime.poll_event_loop(
+                cx,
+                PollEventLoopOptions {
+                    wait_for_inspector: false,
+                    pump_v8_message_loop: false,
```

Comment:
🧐

I guess this option should be set to true. Otherwise, we can't run the tasks V8 has been posted.

Reference: https://groups.google.com/g/v8-users/c/xaSx3ATgFbg/m/8dxCmYZhDgAJ